guide/onboarding/restoring-a-wallet.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: guide
+title: Restoring a wallet
+description: Options for restoring a wallet
+parent: Onboarding
+nav_order: 4
+permalink: /guide/onboarding/restoring-a-wallet/
+main_classes: -no-top-padding
+---
+
+# Restoring a wallet
+
+<div class="center" markdown="1">
+
+There are a number of reasons why a user might need to recover their wallet. Typically, this might be because they lost their device, had it stolen, are migrating from a different wallet or deleted and redownloaded a particular wallet application. Regardless of how it occured, it’s important that your wallet application has clear, easy instructions that provide an easy way for users to recover their funds. There are a number of ways that a wallet recovery can be done. However, as covered in the [wallet interoperability]({{ 'guide/foundations/wallet-interoperability' | relative_url }}) section, not all wallets from one application can be recovered in another.
```

Comment:
```suggestion
There are a number of reasons why a user might need to recover their wallet. Typically, this might be because they lost their device, had it stolen, are migrating from a different wallet or deleted and redownloaded a particular wallet application. Regardless of how it occurred, it’s important that your wallet application has clear, easy instructions that provide an easy way for users to recover their funds. There are a number of ways that a wallet recovery can be done. However, as covered in the [wallet interoperability]({{ 'guide/foundations/wallet-interoperability' | relative_url }}) section, not all wallets from one application can be recovered in another.
```